package.json

```diff
@@ -110,7 +110,7 @@
     "globby": "^13.2.2",
     "got": "^11.8.6",
     "grpc-reflection-js": "^0.3.0",
-    "hookpoint": "3.2.1",
+    "hookpoint": "4.0.0",
```

Comment:
I have built an update for Hookpoint. Now you can see in the Interceptor in the `afterLoop` if the region is disabled ( if hookContext.bail is truthy), because this method is still called then. Say I would still look there to extend this.